src/SimulationRunner.cc

```diff
@@ -463,6 +463,15 @@ void SimulationRunner::PublishStats()
 
   msg.set_paused(this->currentInfo.paused);
 
+  msg.mutable_step_size();
```

Comment:
Do we need this line?

Reply:
this was to create the step size field but fill it with 0s. However, I just simplified to logic to populate the field using `currentInfo.dt` and it should give the right values (0 when paused). [497d8e1](https://github.com/gazebosim/gz-sim/pull/2340/commits/497d8e1253208c87213d01823d1c2764fa134c1b)